contracts/nois-proxy/src/contract.rs

```diff
@@ -723,7 +767,9 @@ mod tests {
 
     const CREATOR: &str = "creator";
 
-    fn setup() -> OwnedDeps<MockStorage, MockApi, MockQuerier> {
+    fn setup(
+        instantiate_msg: Option<InstantiateMsg>,
```

Comment:
Added the ability to pass in a custom `InstantiateMsg`, but if None provided, defaults to the existing hard-coded one.